packages/salesforcedx-vscode-apex/.vscodeignore

```diff
@@ -8,37 +8,20 @@ src/**
 .gitignore
 tsconfig.json
 
-# salesforcedx-utils-vscode files to ignore 
-#node_modules/@salesforce/salesforcedx-utils-vscode/node_modules/**
-node_modules/@salesforce/salesforcedx-utils-vscode/src/**
-node_modules/@salesforce/salesforcedx-utils-vscode/test/**
-node_modules/@salesforce/salesforcedx-utils-vscode/.vscode
-node_modules/@salesforce/salesforcedx-utils-vscode/.vscodeignore
-node_modules/@salesforce/salesforcedx-utils-vscode/package-lock.json
-node_modules/@salesforce/salesforcedx-utils-vscode/tsconfig.json
-node_modules/@salesforce/salesforcedx-utils-vscode/webpack.config.js
-node_modules/@salesforce/salesforcedx-utils-vscode/out/test/**
-
-# salesforcedx-sobjects-faux-generator files to ignore
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/node_modules/**
-!node_modules/@salesforce/salesforcedx-sobjects-faux-generator/node_modules/vscode-nls/**/*
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/src/**
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/test/**
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/.vscode
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/.vscodeignore
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/package-lock.json
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/tsconfig.json
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/webpack.config.js
-node_modules/@salesforce/salesforcedx-sobjects-faux-generator/out/test/**
-
 # ignore unused jsforce code
 node_modules/jsforce/test/**
 node_modules/jsforce/build/**
+node_modules/jsforce/dist/**
 
 # ignore unused sfdx-faye code
 node_modules/sfdx-faye/examples/**
 node_modules/sfdx-faye/site/**
 node_modules/sfdx-faye/spec/**
 
 # ignore unused dayjs code
-node_modules/dayjs/esm/**
\ No newline at end of file
+node_modules/dayjs/esm/**
+
+node_modules/**/rxjs/dist/bundles
+node_modules/**/rxjs/dist/esm
+node_modules/**/rxjs/dist/esm5
+node_modules/**/rxjs/dist/types
```

Comment:
THis is an example of excluding some interior values from an installed node_module. It's horrible, but I don't know a better way due to us being on an older version of a dep that has an old dep on rxjs.